lib/ohai/plugins/linux/platform.rb

```diff
@@ -149,7 +149,7 @@ def platform_family_from_platform(plat)
       "rhel"
     when /amazon/
       "amazon"
-    when /suse/, /sles/, /opensuseleap/, /opensuse/, /sled/
+    when /suse/, /sles/, /opensuseleap/, /opensuse-tumbleweed/, /opensuse-leap-micro/, /opensuse-microos/, /suse-microos/, /opensuse/, /sled/, /sle_hpc/, /sle_rt/
```

Comment:
woudl be nice to be able to remove these 'platform_id_remaps' and add them here instead

opensuse-leap" => "opensuseleap"
sles_sap" => "suse"

looks like this could potentially break if recipes depend on this remapping tho